x/btcstaking/keeper/btc_staking_consumer_events.go

```diff
@@ -40,6 +40,37 @@ func (k Keeper) AddBTCStakingConsumerEvent(ctx context.Context, consumerID strin
 	return nil
 }
 
+func (k Keeper) AddBTCStakingConsumerEvents(ctx context.Context, consumerID string, events []*types.BTCStakingConsumerEvent) error {
```

Comment:
I'd say `AddBTCStakingConsumerEvent` can be removed or simply be a wrapper of this function